module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -572,6 +578,7 @@ public function getHolding($bibId, array $patron = null, array $options = [])
                     'indexes' => $holdingsIndexes,
                     'callnumber' => $holding->callNumber ?? '',
                     'location' => $locationName,
+                    'location_code' => $locationCode,
```

Comment:
I notice that location_code is not a documented field for getHolding(), yet two existing ILS drivers are already using it. I have added it to the spec as an optional field.

Reply:
Thanks! This is what I was hoping you would do. We did this with OLE as well.